OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -39,4 +69,9 @@ public Fragment getFirstFragment() {
         fragment.setArguments(getIntent().getExtras());
         return fragment;
     }
+
+    public void onEvent(CourseUpgradedEvent event) {
+        EventBus.getDefault().removeStickyEvent(CourseUpgradedEvent.class);
```

Comment:
IMO, first consume the event and then remove it.

Reply:
as per discussion, code is equally good in this way